src/index.js

```diff
@@ -27,6 +27,12 @@ export function h(type, props) {
     : type(props || {}, children)
 }
 
+h.ns = {
```

Comment:
@w-mcilhagga What's this?

Reply:
This is the bit where I said:

Defined namespace URIs are added as a property of the h function, for convenience - saves having to look up and type them when doing explicit namespacing.

So if you're doing a foreign object inclusion in svg you can use a canned namespace e.g.

```javascript
h('svg',{},
    h('foreignobject',{width:100, height:100},
        h('p',{namespace:h.ns.html}, 'html included')
    )
)
```

Pure convenience for the end-user to expose the namespace strings via a property attached to h. They have to be defined anyway, so why not give them to the programmer as well.

Up to you if you like this or not. The only advantage to ***not*** exposing them is that you don't need to define the ```html``` namespace in the code.

Reply:
@w-mcilhagga Thanks! Yes, I would probably not expose anything to the user.

Reply:
@w-mcilhagga Ping